sections/tabs.liquid

```diff
@@ -0,0 +1,335 @@
+{{ 'component-accordion.css' | asset_url | stylesheet_tag }}
+{{ 'tabs.css' | asset_url | stylesheet_tag }}
+
+<div class="tabs{% if section.settings.secondary_background == 'none' %} color color-{{ section.settings.color_scheme }}{% endif %}">
+  <div class="page-width">
+    <div class="tabs-wrapper{% if section.settings.secondary_background == 'on_section' %} color color-{{ section.settings.color_scheme }}{% endif %}">
+      <div style="text-align: {{ section.settings.heading_alignment }}">
+        {%- if section.settings.caption != blank -%}
+          <p class="caption-with-letter-spacing"> {{ section.settings.caption }}</p>
+        {% endif %}
+        {%- if section.settings.heading != blank -%}
+          <h2 class="h1">{{ section.settings.heading }}</h2>
+        {% endif %}
+      </div>
+      <div class="grid grid--1-col grid--2-col-tablet tabs__grid{% if section.settings.desktop_layout == 'image_first' %} tabs__grid--reverse{% endif %}">
+        <div class="grid__item">
+          {%- for block in section.blocks -%}
+            <div class="accordion{% if section.settings.secondary_background == 'on_rows' %} color color-{{ section.settings.color_scheme }}{% endif %}" {{ block.shopify_attributes }}>
+              <details id="Details-{{ block.id }}-{{ section.id }}"{% if section.settings.open_first_collapsible_tab and forloop.first %} open{% endif %}>
+                <summary>
+                  <div class="summary__title">
+                    {% render 'icon-accordion', icon: block.settings.icon %}
+                    <h2 class="h4 accordion__title">
```

Comment:
In this tabs section structure we have this new title (optional), but in the product page, these are individual collapsible blocks, would it make sense that both of them are `h3`s? Even when there is no title?

Quick video: https://screenshot.click/01-31-b8giv-5q7ox.mp4

Reply:
Dynamic heading structure can get complex pretty quickly. And we don't want to generate out of order headings.

Thoughts on this approach… for either product accordion or homepage section accordions:

- Always include an `h2` section heading
  - If the merchant supplies a heading title, apply the title within he `h2`
  - If the merchant **does not** supply a title, apply a generic title: `Accordion content` (not `Tab content` as [this will be confusing](https://github.com/Shopify/dawn/issues/906).)
- Accordion item headings will be set as `h3`.

This way the heading structure will be:

1. `h1` page or product title
2. `h2` accordion section title
3.  `h3` accordion item title

cc @katycobb

Reply:
> In the case of the product page, we dont have a higher title and not even an option to add it. In that case would wed always add the title `Accordion content` .

I assumed the two templates shared the same code base. If they're separate components, leave the product section as-is.

And yes, I meant to mention the generic `h2` heading would be set as `visually-hidden` (updated my comment above.) Good catch. 👍

Reply:
> I assumed the two templates shared the same code base. If they're separate components, leave the product section as-is.

Because some of the code is repetitive, I thought of moving it to a re-usable file but that hasnt been done yet 🤔 cc @tyleralsbury 

Here is my original comment: https://github.com/Shopify/dawn/pull/956#discussion_r760313831